File: VB/dxSampleGrid/MyClasses/Person.vb
Namespace dxSampleGrid

    Public Class Person

        Public Sub New(ByVal i As Integer)
            FirstName = "FirstName" & i
            LastName = "LastName" & i
            Age = i * 10
        End Sub

        Public Property FirstName() As String
        Public Property LastName() As String
        Public Property Age() As Integer
        Public Property ParentName() As String
        Public Property IsTabbed() As Boolean
        Public Property IsFloat() As Boolean
    End Class
End Namespace

File: CS/dxSampleGrid/MyClasses/Person.cs
namespace dxSampleGrid {
      
    public class Person  {
       
        public Person(int i) {
            FirstName = "FirstName" + i;
            LastName = "LastName" + i;
            Age = i * 10;
        }

        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int Age { get; set; }
        public string ParentName { get; set; }
        public bool IsTabbed { get; set; }
        public bool IsFloat { get; set; }
    }
}
